.github/workflows/release.yml

```diff
@@ -46,13 +46,26 @@ jobs:
       APPLE_ID_PASSWORD: ${{ secrets.APPLE_ID_PASSWORD }}
       APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID }}
 
+  # ------------------------------------------------------------
+  # 4) Bundle Desktop App (Windows)
+  # ------------------------------------------------------------
+  bundle-desktop-windows:
```

Comment:
Do we want this to be part of the release pipeline right now or a separate experimental pipeline? Sorry, I haven't been following too closely so I am not sure what's the state of this

Reply:
It's fully functional except uvx/npx extension installs